### DISCUSSION TITLE
MSAII%40CMU/

### DISCUSSION THREAD

**Discussion by giscus** [opencsapp/opencsapp.github.io#290] [Announcements]
# MSAII%40CMU/

None

https://opencs.app/MSAII%40CMU/

**Comment by AnnabelleMin** [comment]
24届录取率7-8% 国人里女生显著录的多 同一个本科学校最多两个名额

**Comment by KingArthur0205** [comment]
非常好的项目，可以选AI甚至SCS下面所有的硬课。且11-601 coding bootcamp可以刷题练mock，11-651和17-762也可以练presentation
